### PR TITLE
Include stacktrace failures in incomplete filter

### PR DESCRIPTION
**Pull request by lukaville**
Previously crash filter option only included crash events from LogCat, now it includes stacktrace of test failure as well.

### CODE REVIEW

**Review by jeksor — APPROVED** [review]

File: core/src/main/kotlin/com/malinskiy/marathon/execution/Configuration.kt [hunk 1]
```diff
@@ -121,7 +121,7 @@ data class Configuration constructor(
             testClassRegexes = testClassRegexes ?: listOf(Regex("^((?!Abstract).)*Test$")),
             includeSerialRegexes = includeSerialRegexes ?: emptyList(),
             excludeSerialRegexes = excludeSerialRegexes ?: emptyList(),
-            ignoreCrashRegexes = ignoreCrashRegexes ?: emptyList(),
+            ignoreFailureRegexes = ignoreCrashRegexes ?: emptyList(),
```
Comment (yakimant) [inline core/src/main/kotlin/com/malinskiy/marathon/execution/Configuration.kt]:
```suggestion
            ignoreFailureRegexes = ignoreFailureRegexes ?: emptyList(),
```

File: marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/ConfigurationFactory.kt [hunk 2]
```diff
@@ -107,7 +107,7 @@ private fun createConfiguration(
     testClassRegexes = extensionConfig.testClassRegexes?.map { it.toRegex() },
     includeSerialRegexes = extensionConfig.includeSerialRegexes?.map { it.toRegex() },
     excludeSerialRegexes = extensionConfig.excludeSerialRegexes?.map { it.toRegex() },
-    ignoreCrashRegexes = extensionConfig.ignoreCrashRegexes?.map { it.toRegex(RegexOption.DOT_MATCHES_ALL) },
+    ignoreCrashRegexes = extensionConfig.ignoreFailureRegexes?.map { it.toRegex(RegexOption.DOT_MATCHES_ALL) },
```
Comment (yakimant) [inline marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/ConfigurationFactory.kt]:
```suggestion
    ignoreFailureRegexes = extensionConfig.ignoreFailureRegexes?.map { it.toRegex(RegexOption.DOT_MATCHES_ALL) },
```

**Review by yakimant — CHANGES_REQUESTED** [review]
I guess `ignoreCrashRegexes` shall be renamed everywhere, but not sure. Please check.
That's my first review, don't know the code much.

File: core/src/main/kotlin/com/malinskiy/marathon/execution/Configuration.kt (hunk 1 above)
Comment (lukaville) [reply core/src/main/kotlin/com/malinskiy/marathon/execution/Configuration.kt]: Yes, I missed these ones, will update
Thanks!

**Review by yakimant — APPROVED** [review]

**Pull request merged**